samples/ios-unified/Sample.csproj

```diff
@@ -20,7 +20,7 @@
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
     <ConsolePause>false</ConsolePause>
-    <MtouchArch>i386, x86_64</MtouchArch>
+    <MtouchArch>x86_64</MtouchArch>
```

Comment:
When we dropped iOS 10 support in SDK 12, we no longer had to build for 32-bit systems. Some changes to the SDK build were made to take this into account. Those changes were causing failure in the Xamarin sample app build because it was specifying 32-bit support.

The simulators are all 64-bit at this point.

Reply:
cool